hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```
@@ -643,6 +643,180 @@ objects:
         apiGroup: rbac.authorization.k8s.io
         kind: Role
         name: dedicated-admins-registry-cas-project
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: efs-csi
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+        hive.openshift.io/efs-csi: 'true'
+    resourceApplyMode: Sync
+    resources:
+    - kind: DaemonSet
+      apiVersion: apps/v1
+      metadata:
+        name: efs-csi-node
+        namespace: kube-system
+      spec:
+        selector:
+          matchLabels:
+            app: efs-csi-node
+        template:
+          metadata:
+            labels:
+              app: efs-csi-node
+          spec:
+            nodeSelector:
+              beta.kubernetes.io/os: linux
```

To be on only worker nodes:
```suggestion
              node-role.kubernetes.io/worker: ""
```